codegen/generator/go/templates/src/object.go.tmpl

```diff
@@ -49,12 +61,50 @@ type {{ $field | FieldOptionsStructName }} struct {
 		q: q,
 		c: r.c,
 	}
+
 	{{- else if or $field.TypeRef.IsScalar $field.TypeRef.IsList }}
+		{{- if and $field.TypeRef.IsList (IsListOfObject $field.TypeRef) }}
+    q = q.Select("{{ range $v := $field | GetArrayField }}{{ $v.Name }} {{ end }}")
```

Comment:
super nitpick:

```suggestion
    q = q.Select("{{ range $i, $v := $field | GetArrayField }}{{if $i}} {{end}}{{ $v.Name }}{{ end }}")
```

This should end up with `"foo bar"` instead of `"foo bar "`.

Reply:
Thanks! I was looking for this haha
Fixed in https://github.com/dagger/dagger/pull/4683/commits/2001712417162fca4656a559ddaadebbf73b1274